app/models/concerns/test_track/persistence.rb

```diff
@@ -0,0 +1,29 @@
+module TestTrack::Persistence
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def create!(attributes)
+      new(attributes).tap(&:save!)
+    end
+  end
+
+  def save
+    return false unless valid?
+
+    persist!
+    true
+  rescue Faraday::UnprocessableEntityError
+    errors.add(:base, 'The HTTP request failed with a 422 status code')
+    false
+  end
```

Comment:
Note: The response contains the errors that occurred, and Her would normally assign that to the record. I don't think we're relying on that functionality, so I did the bare minimum here.